### ISSUE TITLE
Podcast Player: Use 24px Sound Icon size

### ISSUE THREAD

**Issue by jeryj** [Automattic/jetpack#15373]
The playing/sound icon is currently at 22px width/height. We should update this to 24px to stay consistent with Gutenberg icon sizes.

Note: There will need to be more adjustments than just changing it to 24px. This increases the height of the playing track by 1px in relation to the rest of the tracks. We'll need to decide how to handle this (setting a min height on the track, setting the height on the wrapping icon span as well, etc).

**Comment by obenland** [comment]
Related: #15337

**Comment by frontdevde** [comment]
While looking into resolving https://github.com/Automattic/jetpack/issues/15337, I was wondering whether this issue https://github.com/Automattic/jetpack/issues/15373 is really an issue and whether it should be addressed.

> We should update this to 24px to stay consistent with Gutenberg icon sizes.

While it is true that the SVG element of most Gutenberg icons has a 24px width/height and viewbox, the visual area of the icons themselves vary quite a bit in size.

<img width="135" alt="Screenshot 2020-04-15 at 11 36 22" src="https://user-images.githubusercontent.com/1562646/79322864-b5d29280-7f0d-11ea-9adc-2d41a852297c.png">

<img width="220" alt="Screenshot 2020-04-15 at 11 36 41" src="https://user-images.githubusercontent.com/1562646/79322894-bf5bfa80-7f0d-11ea-9303-871ac17e9a29.png">

<img width="265" alt="Screenshot 2020-04-15 at 11 36 04" src="https://user-images.githubusercontent.com/1562646/79322923-c84ccc00-7f0d-11ea-930f-ff61dcf5bbcf.png">

<img width="116" alt="Screenshot 2020-04-15 at 11 37 54" src="https://user-images.githubusercontent.com/1562646/79322887-bcf9a080-7f0d-11ea-898c-10d2e26a18f2.png">

The one visual size that seems to occur most often is 18x18px, but there is quite a range across blocks. 


> Note: There will need to be more adjustments than just changing it to 24px. This increases the height of the playing track by 1px in relation to the rest of the tracks. We'll need to decide how to handle this (setting a min height on the track, setting the height on the wrapping icon span as well, etc).

With the above in mind, what would be the benefit of making the icons 24x24, when that doesn't help us with actual visual consistency? Given that this would mean more changes to accommodate for something that isn't visually visible, I'm not sure this is worth a change. Much rather I'd argue we should focus on making sure that the icons for the playing and error states are visually the same size. Which in https://github.com/Automattic/jetpack/pull/15445 they appear to more or less be.

**Comment by WunderBart** [comment]
> While it is true that the SVG element of most Gutenberg icons has a 24px width/height and viewbox, the visual area of the icons themselves vary quite a bit in size.

I really don't think we should be looking into the actual visual area sizes of the icons as they will always vary because each icon is different. I think the important thing here is that they all come from a single source, so keeping the `svg` element size consistent (24px) **will** make the visible area consistent design-wise, regardless of its actual size.

Furthermore, the `SVG` primitive we're using for rendering the element uses 24px as a default value:

<img width="330" alt="Screenshot 2020-04-15 12 10 26" src="https://user-images.githubusercontent.com/1451471/79327073-241a5380-7f14-11ea-89d9-67ca005cb805.png">

...so for me, it feels like we're overriding something for no good reason (which I take the blame for as I set the initial 22px in the first iteration of the block styles where we didn't use the SVG primitive).

---

> > Note: There will need to be more adjustments than just changing it to 24px. This increases the height of the playing track by 1px in relation to the rest of the tracks. We'll need to decide how to handle this (setting a min height on the track, setting the height on the wrapping icon span as well, etc).
>
> With the above in mind, what would be the benefit of making the icons 24x24, when that doesn't help us with actual visual consistency? Given that this would mean more changes to accommodate for something that isn't visually visible, I'm not sure this is worth a change.

There are two things I want to address here. Firstly, I do think that every pixel makes a difference regarding visual consistency. In other words, allowing any deviance (2 pixels in this case) can lead to visual inconsistencies so we should not allow them at all. Secondly, the issue with the 1px height push is something that needs to be addressed separately because it's inconsistent between editor/front-end/themes. For example, it doesn't happen for me on TwentyTwenty (cc @jeryj). I think it's happening for some themes because we're not setting explicit line-height on the track element. I don't think we want it to be inherited because it can e.g. break the icon vertical alignment regardless of its current size. 
To sum up, for me the change is worth the effort! 😄

**Comment by davemart-in** [comment]
I'm not terribly worried about this. I don't think anyone will notice visually at a glance that it's 2px off. I understand the desire to "do things right", but I'm not sure making this change is worth the effort required to ensure that the JS and non-JS versions stay vertically consistent. My vote would be to close this issue, though I'll leave it up to @jeryj to decide since he opened this issue.
